### PR TITLE
fix: display --include-workspace-root option on recursive command's help info

### LINKED ISSUE #5130: Option to run script recursively including root
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

**Describe the user story**

In 6.x `pnpm test -r` runs tests in the root and in subproject (in monorepo repository).

It was very useful to tun all tests in parallel. Now I need to have something like `pnpm test && pnpm test -r`.

**Describe the solution you'd like**

`pnpm test --recursive-with-root` or `pnpm test -R`

**Describe the drawbacks of your solution**

`--recursive-with-root` is long, `-R` is not easy to understand.

**Describe alternatives you've considered**

I thought about reverting 7.x changes with `-r`, but I think 7.x `-r` behavior is more clearly.

### PR DESCRIPTION
**Pull request by await-ovo**
close #5130

### CODE REVIEW

File: pnpm/src/cmd/recursive.ts [hunk 1]
```diff
@@ -102,6 +102,10 @@ Convenient to use in a multi-package repository.',
 A shared lockfile also means that all dependencies of all projects will be in a single node_modules.`,
             name: '--shared-workspace-lockfile',
           },
+          {
+            description: 'When executing commands recursively in a workspace, execute them on the root workspace project as well.',
```
Comment (zkochan) [inline pnpm/src/cmd/recursive.ts]: When there is one sentence in help, we don't put dot in the end.

Comment (await-ovo) [reply pnpm/src/cmd/recursive.ts]: okay

File: pnpm/src/cmd/recursive.ts [hunk 2]
```diff
@@ -77,11 +77,11 @@ and must recompile all your C++ addons with the new binary.',
 
         list: [
           {
-            description: 'Continues executing other tasks even if a task threw an error.',
+            description: 'Continues executing other tasks even if a task threw an error',
             name: '--no-bail',
           },
           {
-            description: 'Set the maximum number of concurrency. Default is 4. For unlimited concurrency use Infinity.',
+            description: 'Set the maximum number of concurrency. Default is 4. For unlimited concurrency use Infinity',
```
Comment (zkochan) [inline pnpm/src/cmd/recursive.ts]: When there are two sentences. I believe we keep the dot.

Comment (await-ovo) [reply pnpm/src/cmd/recursive.ts]: Soory about this, I made the line break a condition for whether to add dot, I'll change it again

**Review by zkochan — APPROVED** [review]

**Pull request merged**